### DISCUSSION TITLE
Allow modification of `fix-version` after issue creation

### DISCUSSION THREAD

**Discussion by BergQuester** [ankitpokhrel/jira-cli#307] [Ideas] [answered]
Currently, `--fix-version` can be set with `jira issue create`.

However, in our workflow we only set the `fix-version` when an issue enters the `Done` status. This means I have to jump into the Jira web app every time move an issue to `Done`. I can imagine other scenarios and workflows where setting `fix-version` after issue creation is helpful.

**Comment by ankitpokhrel** [comment, marked as answer]
@BergQuester This is something on my list and hopefully will work on it soon :)

**Reply by BergQuester** [reply]
@ankitpokhrel Thank you for your response!

And thank you for such a great tool. I had been casting about for a better UI than what Atlassian produces and I stepped up my efforts once the Mac app had been deprecated. Jira-cli has been a joy to use since I found it.
